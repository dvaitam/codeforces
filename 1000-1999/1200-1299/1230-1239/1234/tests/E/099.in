20 6
18 2 18 5 12 19
