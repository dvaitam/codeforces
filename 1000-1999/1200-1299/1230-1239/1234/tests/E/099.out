59 61 57 57 65 55 55 55 55 55 55 65 55 55 55 55 55 22 64 59
